.vimrc

```diff
@@ -84,8 +84,10 @@ set cursorline
 
 ""------------ Mappings --------------"
 "" Create tree navigation
-map <C-n> :vsp . <bar> vertical resize 30 <cr>
+map <C-n> :Lexplore <CR>
```

Comment:
What happened? I've got it working over here.

Em quarta-feira, 14 de outubro de 2015, Rodrigo Siqueira de Melo <
notifications@github.com> escreveu:

> In .vimrc
> <https://github.com/rodrigosiqueira/lappis_vimrc/pull/11#discussion_r41991131>
> :
>
> > @@ -84,8 +84,10 @@ set cursorline
> >
> >  ""------------ Mappings --------------"
> >  "" Create tree navigation
> > -map <C-n> :vsp . <bar> vertical resize 30 <cr>
> > +map <C-n> :Lexplore <CR>
>
> Dude, I tried it here but it's not worked in the same way of the last
> command.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rodrigosiqueira/lappis_vimrc/pull/11/files#r41991131>.
>


-- 
Carlos Oliveira
LAPPIS - UnB/FGA(https://lappis.unb.br/)
GitHub: @carloshfoliveira <https://github.com/carloshfoliveira>
LinkedIn: @carloshenriquef <https://br.linkedin.com/in/carloshenriquef>